Orange/widgets/regression/owunivariateregression.py

```diff
@@ -81,6 +82,7 @@ def __init__(self):
         self.comboBoxAttributesY.setModel(self.y_var_model)
 
         gui.rubber(self.controlArea)
+        self.controlArea.layout().addWidget(self.report_button)
```

Comment:
The button is already added to the layout. This line is needed only if you want to relocate it.